src/main/features/dashboard/views/macro/claimStatus/paidInFullClaimant.njk

```diff
@@ -0,0 +1,39 @@
+{% from "internalLink.njk" import internalLink %}
+
+{% macro paidInFullClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Settling the claim') }}</h2>
+    <p>{{ t('If the defendant has paid you, ' + internalLink('you need to tell us', DashboardPaths.datePaidPage.evaluateUri({ externalId: claim.externalId })) + '.') | safe }}</p>
+{% endmacro %}
+
+{% macro paidInFullClaimantButtonClaimDetails(claim, text='Tell us you’ve settled') %}
+  {% if featureToggles.paidInFull %}
+    <div class="form-group">
+      <p>{{ internalLink(text, DashboardPaths.datePaidPage.evaluateUri({ externalId: claim.externalId }), "button button-secondary") | safe }}</p>
+    </div>
+    {% else %}
+      <div class="form-group">
+        <p>{{ internalLink(text, AppPaths.contactUsPage.uri, "button button-secondary") | safe }}</p>
+      </div>
+  {% endif %}
+{% endmacro %}
+
+{% macro paidInFullClaimantDashboard(claim) %}
+  {{ t('This claim is settled.') }}
```

Comment:
does not make sense to have macro for this small static content.

Reply:
I have discussed this with Rob and we've decided it's best to keep this macro in order to remain consistent.